### DISCUSSION TITLE
Is it possible for someone in the Read or Triage roles, to upload files to a GitHub Project?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#120748] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Projects

### Body

I've got a user who I've put into a GitHub Team, then assigned that team to the GitHub role of Triage. She wants to upload some PDFs which are a part of the software project we're working on, into a GitHub Project. Apparently, she is running into some problems.

So, is it possible for someone who is in the Triage role, to upload files into a GitHub Project? If so, how can we make this happen?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by MindOfBear** [comment]
Hello @rfalanga,

To allow a user with the Triage role to upload files to a GitHub repository go to the repository settings, navigate to "Manage access," and customize permissions to allow the Triage role to push changes.

Now, the user can clone the repository, add PDF files, commit, and push changes.

Have a great day, 
Ovidiu

**Reply by rfalanga** [reply]
I'm sorry, I wasn't clearer. We were thinking of the project manager, the person who is in the Triage role, being able to upload to the GitHub Project, not to the GitHub repository. I had hoped that GitHub Projects would allow uploading document files, like Word or PDF files into the GitHub Project. 

Is that not possible?

**Reply by MindOfBear** [reply, marked as answer]
Oh, now I understand. You can't upload PDFs directly to GitHub Projects, but you can upload them to a repository and then reference them in the project.

To reference a PDF in GitHub Projects, first upload it to your repository. Then, create a card in your project and include a link to the PDF's location in the repository. You can add this link in the card's description, in a note, or as part of an issue or pull request.

This is the only solution.

Kind regards,
Ovidiu

**Reply by rfalanga** [reply]
Oh! That changes everything about what I've been thinking!! I've been writing this while at home, but for my work GitHub account, which I'll reference now: @Rod-at-DOH. 

In my work environment we've got a GitHub organization and we're using our Visual Studio Subscription with GitHub Enterprise licenses to assign people to GitHub Teams, so they can have Write access to the repos we have there. But we didn't want to do that for the project managers. So, I was assigning the PMs to GitHub Teams, then assigning those teams the Triage role. That way I was hoping to have the PMs be only regular users with no write access to any of the repos. But now I've got to rethink this. Technically, @MindOfBear, you've answered the question I asked, but now I've got a different question. I'll create a new question in this forum, using my work GH account, then ask it there.
